proposals/0257-eliminate-vote-fees.md

```diff
@@ -0,0 +1,362 @@
+in leader $l$'s block. $v$ will earn 12 credits based on their stake weight, and
+$l$ will earn 4 credits based on $v$'s stake weight:
+
+$c_v := c_v + 12 s_v$
+
+$c_l := c_l + 4 s_v$
+
+Note that this accounting matches the $M$ derived above, if $v$ votes perfectly
+and includes everyones votes (including their own):
+
+$$c_v = 12 s_v \times 432000 + 4 L \sum_{i\in V} s_i = M$$
+
+In order to implement this change, the vote program will track credits $c_v$ as
+stake weighted totals.
+
+### Vote transaction CU accounting
+
+Vote transactions will no longer count towards the global CU limit in the block,
+they will have their own limit of one vote per validator per block. With SIMD-0218
+IVC, there is no longer any benefit to including multiple old votes within a single
+block, instead only the latest vote will be used.
+
+### Validator set size
+
+By removing the fees for votes, we greatly reduce the barrier for entry as a voter.
+This allows attackers to spin up numerous minimally staked nodes and spam votes
+in an effort to slow down or DoS the cluster.
+
+In order to address this we choose to limit the number of validators that can vote
+at any given time.
+
+These validators that are chosen to vote are the top 2000 validators by stake,
+rotating every epoch.
+
+We must abide by the maximum of 18% of the total cluster stake movement allowed
+per epoch,
+Priority will be given to changes in stake on validators that are already within
+the top 2000 validators by stake before adjustments from inactive validators
+becoming active or active validators becoming inactive.
+
+### TVC window
+
+On the 3rd slot in each validator's sequence of 4 blocks, the timely vote credit
+grace period will only last 1 block. Rewards for landing this vote in the second
+slot will be reduced by 3/8 and follow the normal TVC accounting afterwards.
+
+The reasoning behind this change to avoid scenarios where upcoming leaders bribe
+the voter to delay this vote by 1 block in order for the leader to earn more inclusion
+credits.
+
+## New Terminology
+
+1. $V$ is the set of all validators
+2. $v \in V$ is a validator
+3. $s_v \in [0,1]$ is the stake fraction of validator $v$
+4. $c_v \in [0,1]$ is the stake weighted vote credits of validator $v$
+5. $S = \sum\limits_{v \in V} s_v$ is the total cluster stake
+6. $C = \sum\limits_{v \in V} c_v$ is the total cluster stake weighted vote credits
+7. $I$ is the total inflation rewards issued in an epoch
+8. $T$ represents the total supply of SOL normalized to staked supply of sol
+   $S =1$ so if half of the SOL is staked then $T= 2$
+9. $u_v$ are vote inclusion credits for each validator
+
+## Proofs that the proposed system satisfies the desiderata:
+
+It's clear that maximum validator set size satisfies properties 6 and 7 by
+definition. And that capping entry and exit to 18% satisfies 5, also by
+definition. 1. is satisfied by the removal of vote costs and is the point of the
```

Comment:
When someone writes “by definition,” I want to look up the definition and make sure it’s actually there. In this case, there isn’t even a clear definition. I can guess what you mean, but I don’t want to.